crates/bin/pcli/Cargo.toml

```diff
@@ -48,7 +48,7 @@ tendermint = { version = "0.33.0", features = ["rust-crypto"] }
 jmt = "0.7"
 
 # External dependencies
-ibc-types = { version = "0.6.0", default-features = false, features = ["std", "with_serde"]}
+ibc-types = { version = "0.6.3", features = ["std", "with_serde"] }
```

Comment:
side point, but, i think we should change the `ibc-types` crate to make the feature be called `serde` rather than `with_serde`, since that's a more conventional name

Reply:
@hdevalence Cargo gets confused when using colliding feature names that overlap with an optional dependency, even if you specify `dep:serde`, I never was able to find a way to bundle the `serde = [dep:serde, dep:serde_derive]` etc. maybe it's a skill issue.

Reply:
yeah, i remember it being annoying to figure out. it seems potentially worth it longer-term though.

Reply:
Cargo would not complain about it but the end result is that, somehow, `serde_derive` would not be available to the consumer crate (internal to the ibc-types graph)